Assets/Scripts/Auction/Auctioneer.cs

```diff
@@ -12,6 +12,8 @@ public class Auctioneer : MonoBehaviour
     [SerializeField]
     private AudioSource audioSource;
     [SerializeField]
+    private AudioSource audioSourceFX;
```

Comment:
This is for the kaching sound? Please reconsider the name

Reply:
Will do 👍

Reply:
one should be voicelineSound and the other buySound or sth more actually meaningful but I'm not gonna fite you on dis

Reply:
oh actually do that because you funked up and didn't set the audiosource after rename >:)